stats.py

```diff
@@ -0,0 +1,110 @@
+@@@@@@@@@@@@@@@@/*.,...,##%#**/((##(////(%%%%%%#((/((#%%%%%#*..*/#//@@@@@@@@@@@
+@@@@@@@@@@@@@@@&***,.,./&%#/*****/((///**(#&&%#((((#%%&%##%#.,,,*//%@@@@@@@@@@@
+@@@@@@@@@@@@@@@&,,*/**/#&%/****,*..//,***(%&&(**,,/###%%%(...,,/@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@,,**/**%%%(/(///((((((/**(#%&&&%((*.*(/#%%%(.*/,,(@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@%**,,**#%%#((###(((((/***(#%%&&#(#%&%%&&%(%##,,@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@(,,,(%#####(##%###(//***(#&%&&&&&&&&&&&&&%(/*#@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@/(#*####(((#####((/****/#&&&&&&&&&&&&&&(%%(%@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@(/((((//((###(/*****((#&&%%%%&&&&&&&&##&@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@%//((((///(((/*******/#%&&%%%#(#%&&&&&%%%&@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@&(//**//******,,,,***/(#%%%#*(#%%%%%&&@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@(/***((*********/((#%%%%%&%*/(#%##%&@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@&(/**/((/**  ******/*/((####//#%%#%&@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@&(//**/((//**,.*/#%%%%%%,/%%##%%%%&@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@&((/***/(//***//(((((((#%&&%%%%%%%@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@&(((/***////**////((##%%&&&%%%%#%@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@((((/***/(//////(((##%%%&&%%%##&@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@(((((/////(#(######%%%&&&%%##(&@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@(((#(//////(##%%%%%&&&&&%##(#%@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@#((#((////**/##%%%&&&&%%#((%%%@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@#((##(/*******/((#####(((#%%%%&@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@%((##(/***,,,,,,,,,*/(##%%%%&%#%%(*//(%@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@%###((((/********//((##%%%%&&&&%%%%#****(//%@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@##(#((#((((/****///(((##%%%%%%%&&%%%%#********//(#@@@@@@@@@@@
+@@@@@@@@@@@@((((((((//#(((//****//(((##%%%%%%%%%%%%%(**,**/****///(((&@@@@@@@
+@@@@@@@@@%#(((((((///(//#(////****//((###%%%%%%%%%%%#*,,,**/***////////((((#%@@
+@@@@@%#(*/((((((//**////%#///****//(((#####%%%%%%%*,,,,***/*******/////////(((#
+@@@#((/////////((((((/**##(/*****///(((((######(,,,,,**/*****/////*******//////
+@%((///////*****//(/((/**(((*******///((((((/,,,,,,***************,*/********//
+@(/*///************//((**,*//*****/////*,,...,,,,*******,,,,,,,,,,,,,,,**/*****
+%/*****/***********,,*//*,,,.   .........,,,,****,,,,,,,,,,,,,,,,,,,,*****,*//*
+(,.,*****///******/*********,.......,,,,*,,,,,,,,,,,,,,,,,,,,,,,,,,,****,,,,,**
+/, .,********//*********,,,,..,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,**,,,,,,,,,**
+/.  .**********/*******,,,,,,,,.....,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,.,**
+
+                  "We do not sell data to advertisers"
+                  "Violating your privacy since 2004™"
+                             - The Zucc
+'''
+
+import json
+
+
+def get_stats():
+    """Returns the current stats from the file as a JSON object"""
+    try:
+        with open('stats.json') as f:
```

Comment:
Could you move this to config, please? I think we should create a database folder where all folders that contain data should be stored.

Reply:
Why though? It's rather unnecessary, don't you think?

Reply:
Well, may there ever come an event in the future when wishes to rename the file or move it to a different location, I can imagine they'd prefer to have it all in one location, don't you think?